packages/cbioportal-ts-api-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-ts-api-client",
   "description": "cBioPortal API Client for TypeScript",
-  "version": "1.0.0-beta.1",
+  "version": "0.9.0-beta.0",
```

Comment:
This is to prevent lerna from auto publishing version `1.0.0`. When this PR gets merged, version `0.9.0` will be published instead.